index.html

```diff
@@ -8,28 +8,43 @@
     <script src="https://kit.fontawesome.com/d628e85c59.js" crossorigin="anonymous"></script>
     <title>Andisiwe Portfolio</title>
 </head>
-<body class="portfolio">
-                                        <!--Task-bar-->
+<body class="portfolio">                                        <!--Task-bar-->                        
     <header>
         <nav class="nav">
             <div class="container">
                 <div> 
-                    <a class="Mylogo" href="#"> Andisiwe</a>
+                    <a class="Mylogo" href="#">Andisiwe</a>
```

Comment:
It essentially looks good, but as mentioned by the previous reviewer it's recommended to follow the exact Figma design. So please make sure to fix the sizing and the spacing issue.

![image](https://user-images.githubusercontent.com/34779928/194695081-89ca46d5-b8e4-41fe-ba8d-c2b213d7cbd9.png)